roles/reset/tasks/main.yml

```diff
@@ -67,6 +67,17 @@
   tags:
     - docker
 
+- name: reset | stop all cri-o containers
+  shell: "crictl ps -aq | xargs -r crictl stop"
```

Comment:
please provide path to crictl. ```{{ bin_dir }}/crictl```

Reply:
crictl is not kept at the default bin_dir location `(/usr/local/bin)` and is installed from the os package. for eg: in the case of clearlinux, it is in `/usr/bin` and that might cause issues.

Reply:
There is another path `docker_bin_dir` which i think can be expected to the path of system package installed files. we could change that to a more generic name and use that as the prefix for both docker and crio, if that would work.